src/Apps/Artist/Routes/Overview/index.tsx

```diff
@@ -297,7 +297,13 @@ export class OverviewRoute extends React.Component<OverviewRouteProps, {}> {
             </Media>
             <Row>
               <Col>
-                <SectionHeader headerString="Works For Sale" />
+                <SectionHeader
+                  headerString={
+                    artist.counts.forSaleArtworks
```

Comment:
```suggestion
                    artist.counts.forSaleArtworks > 0
```

It's a really minor thing, but we're explicit about the count comparison in other places which could be mirrored here. 

I don't consider this a required change, I'll leave it up to you.